python/kvikio/nvcomp.py

```diff
@@ -0,0 +1,235 @@
+import collections
+
+import cupy as cp
+import Enum
```

Comment:
Appears to be causing [a CI build failure]( https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/kvikio/job/prb/job/kvikio-cpu-cuda-build/CUDA=11.5/95/console )

```suggestion
from enum import Enum
```